R/pkg/R/generics.R

```diff
@@ -695,10 +695,6 @@ setGeneric("desc", function(x) { standardGeneric("desc") })
 
 #' @rdname column
 #' @export
-setGeneric("endsWith", function(x, ...) { standardGeneric("endsWith") })
```

Comment:
I kept this definition before. But sparkR complains no signature found. The tests only work if I remove it in generic.R. Let me try again.